include/dts-environment.sh

```diff
@@ -181,6 +181,8 @@ declare HEADS_SWITCH_FLASHROM_OPT_OVERRIDE
 declare PLATFORM_SIGN_KEY
 
 # Other variables:
+# dts-scripts config path
+declare DTS_CONFIG_REF="${DTS_CONFIG_REF:-main}"
```

Comment:
```suggestion
declare DTS_CONFIG_REF="${DTS_CONFIG_REF:-/refs/heads/main}"
```

that way we could also download by commit instead of branch: https://docs.github.com/en/repositories/working-with-files/using-files/downloading-source-code-archives#source-code-archive-urls

Reply:
@m-iwanicki sure, improved: [-f diff](https://github.com/Dasharo/dts-scripts/compare/22adb9a3c8af4ee2ba423748ae1d9e3d8b1a3ce0..ac6af147a3d4dcdbfae284010833d26804ff844d)